rdt/transformers/base.py

```diff
@@ -52,6 +164,28 @@ def reverse_transform(self, data):
             data (pandas.Series or numpy.array):
                 Data to transform.
 
+        Returns:
+            pandas.Series:
+                Reverted data.
+        """
+        output_types = self.get_output_types()
+        output_columns = list(output_types.keys())
+        columns_data = data[output_columns]
+        reversed_columns = self._reverse_transform(columns_data)
+        data[self._columns] = reversed_columns
+
+        columns_to_drop = set(output_columns) - set(self._columns)
```

Comment:
This could possibly be captured inside the `fit` method, after calling `self._fit`, since at that time the output names are already known.

Reply:
I don't understand the reason for this change? It makes sense for the logic of figuring out which columns to drop to be in the reverse_transform, since that's where we drop the columns. Since we only drop the columns in the reverse_transform, it seems unnecessary to create an extra attribute in a different function, since computing the `columns_to_drop` is trivial.